airflow/sensors/base_sensor_operator.py

```diff
@@ -176,3 +179,60 @@ def deps(self):
         if self.reschedule:
             return BaseOperator.deps.fget(self) | {ReadyToRescheduleDep()}
         return BaseOperator.deps.fget(self)
+
+
+def poke_mode_only(cls):
```

Comment:
Why do we need such a complex decorator?
https://github.com/apache/airflow/pull/7197#discussion_r419218559